### PR TITLE
Replace parallel morning challenge with waterfall morning challenge #834

### LINKED ISSUE #834: Replace parallel morning challenge with waterfall morning challenge

### PR DESCRIPTION
**Pull request by idantb101**
Week 3 - Curriculum Planning
 relates #824
close #834

### CODE REVIEW

**Comment by astroash** [comment]
☝️ Can you remove closing of `week-epic` #824? Your PRs should only close the individual issue created from the epic 😊

**Comment by idantb101** [comment]
Is it ok know? , 
I meant only to relate the issue
@astroash

**Comment by idantb101** [comment]
fixed, thank for the input :+1:

File: curriculum-planning/resource-index.md [hunk 1]
```diff
@@ -67,4 +67,7 @@
 * [Handlebars](https://github.com/Jbarget/handlebars-morning-challenge)\*
 * [Recursion](https://github.com/foundersandcoders/mc-recursion)
 
+(https://github.com/foundersandcoders/mc-parallel-functions)\*
```
Comment (astroash) [inline curriculum-planning/resource-index.md]: This is not an external repo so doesn't need a \* 🌟

**Comment by m4v15** [comment]
Looks good - @trc5019 @parkhabit, just tagging you guys so you're notified of the change for your mentor prep for week 3.

**Pull request merged**